src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentCost = 40;
+  const sum = days * rentCost;
+  const bigDiscount = 50;
+  const smallDiscount = 20;
+
+  if ((days > 2) && (days < 7)) {
+    return sum - smallDiscount;
+  } else if (days > 6) {
+    return sum - bigDiscount;
+  }
```

Comment:
Checklist item 3

Reply:
Is it npm run lint?

Reply:
Link to the task checklist is here:
![image](https://user-images.githubusercontent.com/60885495/189907400-4c8af42a-1fb3-46ba-a841-b791ed910999.png)

Also, it's `checklist.md` file in the task repository.

Reply:
:) I didn't know that was there